src/wallet/wallet.cpp

```diff
@@ -592,7 +754,23 @@ bool CWallet::EncryptWallet(const SecureString& strWalletPassphrase)
             // die and let the user reload their unencrypted wallet.
             assert(false);
         }
+        if (!hdChainCurrent.IsNull()) {
+            assert(EncryptHDChain(vMasterKey));
+
+            CHDChain hdChainCrypted;
+            assert(GetHDChain(hdChainCrypted));
+
+
+                LogPrintf("EncryptWallet -- current seed: '%s'\n", HexStr(hdChainCurrent.GetSeed()).c_str());
```

Comment:
indent and blank lines nit

Reply:
shouldn't those be `printf` inside a `DBG()` macro?

Reply:
yes,i had used logprintf it when i couldn't find the `DGB()` macro,ill add it back to use `DBG()` instead